app/components/slide_up_panel_item/index.tsx

```diff
@@ -15,6 +15,7 @@ import {isValidUrl} from '@utils/url';
 type SlideUpPanelProps = {
     destructive?: boolean;
     icon?: string | Source;
+    rightIcon?: boolean;
```

Comment:
Quick solution to set the button to the right side. Not sure if the desirable one.

Reply:
where in the design we have a right icon in a slide up panel item as I could not find it. Also would changing it from left to right not cause the UI to be different than the rest, is this expected?

Also this can be accomplish with adding an additional style instead of duplicating the component

Reply:
You can see the prototype here:
https://www.figma.com/proto/o5tZwjMRm5JyLkQUWhFkIg/Browse-channels-flow?page-id=5%3A10194&node-id=10%3A8346&viewport=354%2C48%2C0.5&scaling=min-zoom&starting-point-node-id=10%3A8344&show-proto-sidebar=1

Regarding updating the position through the style... do you mean using something like reversing the order of the flex? Or something else?